components/desktop/mate/mate-screensaver/Makefile

```diff
@@ -0,0 +1,80 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL)". You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2022 Carsten Grzemba
+#
+
+MATE_CATEGORY= APP
+BUILD_STYLE= configure
+include ../../../../make-rules/shared-macros.mk
+include $(WS_MAKE_RULES)/mate.mk
+
+COMPONENT_NAME=		mate-screensaver
+COMPONENT_VERSION=	1.26.1
+COMPONENT_MJR_VERSION=	1.26
+COMPONENT_MNR_VERSION=	1
+COMPONENT_SRC=	$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=	$(COMPONENT_SRC).tar.xz
+COMPONENT_ARCHIVE_HASH=	\
+	sha256:4fbdb21ea4a59ea8de33ea9bf776d869753e49295604664c30e220e09659b8bc
+COMPONENT_ARCHIVE_URL=	https://pub.mate-desktop.org/releases/$(COMPONENT_MJR_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL= https://pub.mate-desktop.org
+COMPONENT_CLASSIFICATION=	Applications/System Utilities
+COMPONENT_LICENSE= GPLv3
+COMPONENT_LICENSE_FILE= COPYING
+COMPONENT_FMRI= desktop/mate/$(COMPONENT_NAME)
+COMPONENT_SUMMARY= Mate Screensaver
```

Comment:
COMPONENT_FMRI is also set in mate.mk,
mate.mk has been introduced to set most of the variables automatically.